src/pages/press.js

```diff
@@ -30,7 +31,8 @@ const Press = () => {
             </p>
             <p className='body'>
               Full story:{' '}
-              <Link to='/press/2022-04-21-civicactions-announces-Ryerson-partnership-digital-accessibility'>
+              <Link to='/press/2022-04-21-civicactions-announces-Ryerson-partnership-digital-accessibility'
+              className='external-link'>
```

Comment:
This one looks like an internal link so it shouldn't get the class.

Reply:
Nice catch. Fixed